app/main/posts/views/card.html

```diff
@@ -14,7 +14,7 @@
             <div class="postcard-title">
                 <div class="postcard-field">
                 <div class="tweet-container">
-                  <twitter-widget ng-if="post.data_source_message_id" twitter-widget-id="post.data_source_message_id">
+                  <twitter-widget ng-if="post.data_source_message_id && post.source === 'twitter'" twitter-widget-id="post.data_source_message_id">
                   </twitter-widget>
```

Comment:
Keeping the message_id check, because if no message_id we cannot render the tweet anyway...